### ISSUE TITLE
OB Cannon not Firing

### ISSUE THREAD

**Issue by Harrysno** [cmss13-devs/cmss13#6357]
### Testmerges

#6188, #6071, #6304, #6039, #5513

### Round ID

22462

### Description of the bug

Coordinates were typed into the Orbital Bombardment Section on Overwatch Consoles as normal. When the "Fire" button was pressed the OB would not fire, no error message was given such as "Roof Too Thick" or "Biological Protection." It simply would not fire. This issue persisted across multiple different coordinates and multiple Overwatch Consoles (I tested it on Delta & Charlie consoles). Loaded Warhead was HE with 5 Fuel Blocks.

### What's the difference with what should have happened?

OB was fueled correctly, chambered properly, and dialed in on eligible coordinates. It did not fire though, which isn't right.

### How do we reproduce this bug?

1. Load OB (HE) into the cannon and fuel it (5 Blocks), then chamber.
2. Dial OB Coordinates into Overwatch Console and click "Fire." Given coordinates were "0, -78" and "-2, -61"
3. Watch as nothing occurs.
...

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by Harrysno** [comment]
Figured it out, was a TGUI issue. Hitting F5 to refresh the menu fixed the issue. :^)

**Comment by Drulikar** [comment]
If you're able to reproduce this it should still be an open issue. But if it no longer occurs for you, maybe it was just a bad cache.

**Comment by Harrysno** [comment]
Fair enough, I didn't entirely consider that it might have implications beyond the immediate round. Some important context that I realize I forgot to include is that **nobody** was able to properly get the OB to fire prior to refreshing the UI, not just an issue I was experiencing.
